src/ralph/data_center/apps.py

```diff
@@ -0,0 +1,9 @@
+from ralph.apps import RalphAppConfig
+
+
+class DataCenterConfig(RalphAppConfig):
+    name = 'ralph.data_center'
+    verbose_name = 'Data Center Assets'
+
+    def get_proxies(self):
+        return ['DCHost']
```

Comment:
I choose this solution because i have a problem when i added permission for all proxy models - some pair (app_label, codename) in permission table was duplicated and then ``sitetree`` raised ``MultipleObjectsReturned``.